packages/manager/apps/pci-savings-plan/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -45,20 +49,28 @@ const Breadcrumb: React.FC = () => {
     updateNav();
   }, [navigation, projectId]);
 
+  const breadcrumbItems = [
+    {
+      href: urlProject,
+      label: project?.description,
+    },
+    {
+      href: `${urlProject}/savings-plan`,
+      label: t('title'),
+    },
+    ...items,
+  ];
+
   return (
-    <OsdsBreadcrumb
-      items={[
-        {
-          href: urlProject,
-          label: project?.description,
-        },
-        {
-          href: `${urlProject}/savings-plan`,
-          label: t('title'),
-        },
-        ...items,
-      ]}
-    ></OsdsBreadcrumb>
+    <OdsBreadcrumb>
+      {breadcrumbItems.map((item) => (
+        <OdsBreadcrumbItem
+          key={item.href}
```

Comment:
I think the key here should be the index because I'm not sure if 2 items could have the same href